src/components/VideoPlayer/BaseVideoPlayer.tsx

```diff
@@ -233,7 +235,18 @@ function BaseVideoPlayer({
         (event: VideoFullscreenUpdateEvent) => {
             onFullscreenUpdate?.(event);
 
+            if (event.fullscreenUpdate === VideoFullscreenUpdate.PLAYER_DID_PRESENT) {
+                // When the video is in fullscreen, we don't want the scroll to be captured by the InvertedFlatList of report screen.
+                // This will also allow the user to scroll the video playback speed.
+                if (videoPlayerElementParentRef.current && 'addEventListener' in videoPlayerElementParentRef.current) {
+                    videoPlayerElementParentRef.current.addEventListener('wheel', stopWheelPropagation);
+                }
+            }
```

Comment:
I modified the solution a bit so we only register when the full screen is present. This way, we won't have a listener for each video player.